### PR TITLE
[DDW-756] Fix popover overlap

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
This PR fixes the overlapping popover arrow that blocked outside-click events.

## Todos

- [x] Switch to a stable release of react-polymorph after testing

## Screenshots

Please refer to the [react-polymorph PR](https://github.com/input-output-hk/react-polymorph/pull/210).

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1649774529121849)
- [ ] Test

---

### Basics

- [x] PR assigned to the PR author(s)
- [x] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [x] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [x] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [x] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [x] PR link is added to a Jira ticket, ticket moved to In Review
- [x] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [x] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [x] Storybook works and no stories are broken (`yarn storybook`)
- [x] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,13 +160,6 @@ Most of the commands need `nix` and will run only on Linux or macOS.
 - Windows 10/11
 - Daedalus testnet installation (similar version used in branch) in `C:\Program Files\Daedalus Testnet`
 - NodeJS 16
-- Python2 *
```
Comment (renanvalentin) [inline README.md:163]: These instructions is needed in case you want to run Daedalus on windows

**Review by renanvalentin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (przemyslaw-wlodek) [reply README.md:163]: Hey, thanks 😄  I've just discussed with @danielmain and he's going to test that on Windows. Most probably it's no longer needed (as it was marked already* as node-sass dependency, which I've removed).

**Review by szymonmaslowski — APPROVED** [review]

**Review by alexander-rukin — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (przemyslaw-wlodek) [reply README.md:163]: We've decided to leave it as is for now (just without mentioning `node-sass` explicitly).

**Review by gabriela-ponce — APPROVED** [review]

**Review by ManusMcCole — APPROVED** [review]

**Review by renanvalentin — APPROVED** [review]

**Pull request merged**